test/parallel/test-zlib-create-raw.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const zlib = require('zlib');
+
+{
+  const inflateRaw = zlib.createInflateRaw();
+  assert(inflateRaw instanceof zlib.InflateRaw);
+}
+
+{
+  const deflateRaw = zlib.createDeflateRaw();
+  assert(deflateRaw instanceof zlib.DeflateRaw);
+}
```

Comment:
You mean the scoping braces? No, not strictly necessary, but desirable IMO. Having every test in its own scope means (among other benefits) that you can freely add and remove variables in one test without worrying about side effects in any of the other tests. Right now, there's only two tests in this file but ideally more tests will be added over time.